tests/rptest/remote_scripts/topic_operations.py

```diff
@@ -0,0 +1,376 @@
+import argparse
+import json
+import logging
+import numpy
+import random
+import string
+import sys
+import time
+
+from concurrent.futures import ThreadPoolExecutor
+from copy import deepcopy
+
+from kafka import KafkaAdminClient
+from kafka.errors import NoBrokersAvailable
+from kafka.admin import NewTopic
+
+
+def setup_logger():
+    handler = logging.StreamHandler()
+    handler.setFormatter(
+        logging.Formatter(
+            '%(asctime)s - %(name)s - %(levelname)s - %(message)s'))
+    handler.setLevel(logging.DEBUG)
+    logger = logging.getLogger("SerdeClient")
+    logger.addHandler(handler)
+
+    return logger
+
+
+def write_json(ioclass, data):
+    ioclass.write(json.dumps(data))
+    ioclass.write('\n')
+    ioclass.flush()
+
+
+class TopicSwarm():
```

Comment:
fun!

Reply:
The goal was to build isolated tool with no complexity in mind.